RELEASE.md

```diff
@@ -14,8 +14,9 @@ Example `version`
 
 - Update and review `CHANGELOG.md`:
 
-      gem install github_changelog_generator
-      github_changelog_generator -u scientific-python -p spin --since-tag=<last tag>
+      changelist scientific-python/spin  <v0.0> main --version <0.1> >> CHANGELOG.md
```

Comment:
```suggestion
      changelist scientific-python/spin v0.0 main --version v0.1 > CHANGELOG.md
```

Not sure if that should be v0.1 or 0.1

Reply:
We can take out the angle brackets indicating what needs to be changed. But we need to leave in the `>>`.

Reply:
If you take out the angle brackets, we should make it clear that that text needs to be changed in another way. Maybe just explain it in the text below.

Reply:
An argument for keeping the angle brackets is that is the standard notation for indicating a manadatory argument that needs to be replaced when using.

Reply:
Maybe you would prefer
```
      changelist scientific-python/spin  <tag of the previous release> main --version <upcoming release version> >> CHANGELOG.md
```

Reply:
It'd be nice to be able to copy, paste, and modify. The brackets are a bit noisy and hard to read.